server/room.ts

```diff
@@ -51,6 +51,7 @@ export class Room {
   // Non-serialized state
   public roomId: string;
   public roster: User[] = [];
+  public subscribers: BooleanDict = {};
```

Comment:
hmm, why don't we just add a property to each user in roster?

Reply:
> hmm, why don't we just add a property to each user in roster?

I actually had done it that way at first but changed it because nameMap works the same way and just to have simpler access.

Do you think it would be better if  we added  a property to each user in the roster array?